plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -139,6 +139,9 @@
       <consolePageParticipant
             class="com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsolePageParticipant"
             id="com.google.cloud.tools.eclipse.appengine.consolePageParticipant">
+              <enablement>
+                <instanceof value="com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsolePageParticipant"/>
```

Comment:
This seems weird: check that the object is an instance of itself?  Should this be a `LocalAppEngineConsole`?